packages/mux-elements-codemod/package.json

```diff
@@ -67,7 +66,6 @@
     "rules": {
       "no-shadow": "error",
       "no-extra-boolean-cast": 0,
-      "@mux/i18n/no-substitutions": "error",
```

Comment:
question(non-blocking): Why were these changes introduced in this PR?

Reply:
it's a rule that is not needed in that pkg, I feel like if it's not done when you notice small things like that it will never get done so did it "out of scope"